modules/appeals_api/app/controllers/appeals_api/metadata_controller.rb

```diff
@@ -54,13 +62,21 @@ def appeals_status_upstream_healthcheck
                               upstream_service_details(service, health_checker, time)
                             end
         }
-      }, status: health_checker.appeals_services_are_healthy? ? 200 : 503
+      }, status: appeals_status.call
     end
 
     def decision_reviews_upstream_healthcheck
       health_checker = AppealsApi::HealthChecker.new
       time = Time.zone.now.to_formatted_s(:iso8601)
 
+      decision_reviews_status = lambda {
+        begin
+          health_checker.decision_reviews_services_are_healthy? ? 200 : 503
+        rescue
```

Comment:
Log the exception & message here, too. We usually prefix the messages with something easily searchable like `AppealsApi` or `AppealsApi::Healthchecker`.

Reply:
added 👍